aea/cli/create.py

```diff
@@ -75,7 +75,7 @@ def create(click_context, agent_name):
         # create a config file inside it
         logger.info("Creating config file {}".format(DEFAULT_AEA_CONFIG_FILE))
         config_file = open(os.path.join(agent_name, DEFAULT_AEA_CONFIG_FILE), "w")
-        agent_config = AgentConfig(agent_name=agent_name, aea_version=aea.__version__, author="", version="v1", license="", fingerprint="", url="", registry_path=DEFAULT_REGISTRY_PATH, description="")
+        agent_config = AgentConfig(agent_name=agent_name, aea_version=aea.__version__, author="", version="1.0.0", license="", fingerprint="", url="", registry_path=DEFAULT_REGISTRY_PATH, description="")
```

Comment:
I think the default version should be `0.1.0`. We should not suggest that a default agent project is somehow a v1.

Reply:
done